docs/attestation/intel/sgx/sgx_process.msc

```diff
@@ -0,0 +1,31 @@
+# Commits which modify this file MUST generate the new .png!
+# .png files can be generated at https://mscgen.js.org/
```

Comment:
Our automation will enforce this rule if appropriate entries are added to [docs/Makefile](https://github.com/enarx/enarx/blob/master/docs/Makefile#L1).

The automated script depends on this relationship between the markup and the diagram filename:

PNG: `docs/attestation/intel/sgx/sgx_certchain.msc.png`
MSC: `docs/attestation/intel/sgx/sgx_certchain.msc`

so for it to work for these diagrams out of the box you'll need to change the file extension of the png you checked in to end with `.msc.png` instead of just `.png`

Reply:
Thanks for pointing this out, @connorkuehl . I changed the file names and included them in the doc Makefile.

Reply:
Actually, the old ones are still here for some reason even though I did `git rm`. Will fix before I ping for re-review.